docs/usage.md

```diff
@@ -27,14 +27,14 @@ Navigate to demo alert and note the following:
     - Next steps
 
 
-- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs following actions.
+- The playbook `Fortinet FortiNDR Cloud > Fetch Detection Events` runs manually and performs the following actions.
 
     - Fetch events for specified detection and append the same to the source data in the alert.
 
-    - Creates a record for the asset on which data exfiltration issue detected and links the same to the alert.
+    - Creates a record for the asset on which the data exfiltration issue was detected and links the same to the alert.

```

Comment:
Suggestion: Creates a record for an asset on which data exfiltration issue is detected and link it to an alert.

Reply:
Update this as below:
Creates a record for an asset on which the data exfiltration issue is detected and links it to the alert.